src/app/clients/ActionBar.cy.tsx

```diff
@@ -9,34 +9,39 @@ describe("Clients - Action Bar", () => {
         {
             primaryKey: "primaryKey1",
             addressPostcode: "AB1 2CD",
-            collectionCentre: "Centre 1",
+            deliveryCollection: {
+                collectionCentre: "Centre 1",
+                congestionChargeApplies: false,
+            },
             collectionDatetime: new Date().toISOString(),
-            congestionChargeApplies: false,
             familyCategory: "Family 1",
-            flaggedForAttention: false,
             fullName: "John Smith",
-            lastStatusUpdate: new Date().toISOString(),
             lastStatus: "Delivered",
             packingDatetime: new Date().toISOString(),
             packingTimeLabel: "10:00",
             parcelId: "123456789",
-            requiresFollowUpPhoneCall: false,
+            iconsColumn: {
+                requiresFollowUpPhoneCall: false,
+                flaggedForAttention: false,
+            },
+            voucherNumber: "123456789",
         },
         {
             primaryKey: "primaryKey2",
             addressPostcode: "AB1 aaaa2CD",
-            collectionCentre: "Centraaaae 1",
+            deliveryCollection: { collectionCentre: "Centraaaae 1", congestionChargeApplies: true },
```

Comment:
How come congestionChargeApplies isn't just optional props?

Reply:
Also what was wrong with `Centre 2`?

Reply:
Nothing, no thought went into that because the name doesn't really matter.

Congestion charge is required for the column as the red C icon is displayed in that column